py/visdom/server.py

```diff
@@ -1743,6 +1742,32 @@ def download_scripts(proxies=None, install_dir=None):
                 logging.error('Error {} while downloading {}'.format(
                     exc.reason, key))
 
+    # Download MathJax Js Files
+    import requests
+    cdnjs_url = 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/' 
+    mathjax_dir = os.path.join(*cdnjs_url.split('/')[-3:])
+    mathjax_path = [
+        'config/Safe.js?V=2.7.5',
+        'config/TeX-AMS-MML_HTMLorMML.js?V=2.7.5',
+        'extensions/Safe.js?V=2.7.5',
+        'jax/output/SVG/fonts/TeX/fontdata.js?V=2.7.5',
+        'jax/output/SVG/jax.js?V=2.7.5',
+        'jax/output/SVG/fonts/TeX/Size1/Regular/Main.js?V=2.7.5',
+        'jax/output/SVG/config.js?V=2.7.5',
+        'MathJax.js?config=TeX-AMS-MML_HTMLorMML%2CSafe.js&#038;ver=4.1',
+    ]
+    mathjax_dir_path = '%s/static/%s/%s' % (install_dir, 'js',mathjax_dir)    
+    
+    for path in mathjax_path:
+        filename=path.split("/")[-1].split("?")[0]
+        extracted_directory=os.path.join(mathjax_dir_path,"\\".join(path.split('/')[:-1]))
```

Comment:
As a note for good coding practice, I won't always point out every location that has an issue if I've noted it elsewhere in the file. `"\\".join(<list>)` should be `os.path.join(*<list>)` as above.

Reply:
oops.. I am really sorry. I neglected it. Thank you. I will check it out.

Reply:
There are no `'\\'.join(<list>)` now.